projects/1.18/assets/create-clockwork/vs_clockwork/lang/zh_cn.json

```
@@ -0,0 +1,129 @@
+{
+  "itemGroup.clockwork": "VS: 发条装置",
```

```suggestion
  "itemGroup.clockwork": "VS：发条装置",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。